src/graphics/reproject-texture.js

```diff
@@ -78,8 +82,16 @@ function reprojectTexture(source, target, options = {}) {
     const specularPower = options.hasOwnProperty('specularPower') ? options.specularPower : 1;
     const numSamples = options.hasOwnProperty('numSamples') ? options.numSamples : 1024;
```

Comment:
should we default to less samples now?

Reply:
Don't think so. 1024 samples is a good choice for some pathological cases. Also, this numSamples still applies to phong specular, which isn't importance sampled.